net/mwan3/files/etc/hotplug.d/iface/15-mwan3

```diff
@@ -66,11 +66,14 @@ case "$ACTION" in
 		mwan3_set_iface_hotplug_state $INTERFACE "$status"
 		if [ "$MWAN3_STARTUP" != "init" ]; then
 			mwan3_create_iface_route $INTERFACE $DEVICE
-	                mwan3_set_general_rules
+			mwan3_set_general_rules
 			[ "$status" = "online" ] && mwan3_set_policies_iptables
 		fi
 		[ "$ACTION" = ifup ] && procd_running mwan3 "track_$INTERFACE" && procd_send_signal mwan3 "track_$INTERFACE" USR2
 		;;
+	ifupdate)
+		[ "$ACTION" = ifup ] && procd_running mwan3 "track_$INTERFACE" && procd_send_signal mwan3 "track_$INTERFACE" USR2
```

Comment:
Does this really work? 
The ACTION is ifupdate but we check `[ "$ACTION" = ifup ]`

Reply:
Yes, looks like it would not really work. This is a real issue, but I haven't been able to recreate it or test. I updated the P/R, but perhaps we should close this for now until another user runs into it and can test the fix.